package/MDAnalysis/analysis/align.py

```diff
@@ -630,7 +630,8 @@ def rms_fit_trj(
         tol_mass=0.1,
         strict=False,
         force=True,
-        quiet=False,
+        verbose=None,
+        quiet=None,
```

Comment:
I moved the quiet keyword at the end of the signatures when it was not already. See f5f3e798c385d538fe8cfbf9306336e1eb6a6a0c